var/spack/repos/builtin/packages/namd/package.py

```diff
@@ -19,6 +19,7 @@ class Namd(MakefilePackage):
     manual_download = True
 
     version("develop", branch="master")
+    version('2.14b2', sha256='cb4bd918d2d545bb618e4b4a20023a53916f0aa362d9e57f3de1562c36240b00')
     version('2.14b1', sha256='9407e54f5271b3d3039a5a9d2eae63c7e108ce31b7481e2197c19e1125b43919')
     version('2.13', '9e3323ed856e36e34d5c17a7b0341e38')
```

Comment:
I assume 2.14b1 and 2.14b2 are beta releases? We don't usually like to install beta releases by default. I would use:
```suggestion
    version('2.13', '9e3323ed856e36e34d5c17a7b0341e38', preferred=True)
```
Not that it really matters since the source code must be manually downloaded, so there really is no "default".